tasks/security_agent.py

```diff
@@ -292,7 +290,7 @@ def build_embed_syscall_tester(
     nf_path = os.path.join(ctx.cwd, 'syscall-tester.ninja')
     with open(nf_path, 'w') as ninja_file:
         nw = NinjaWriter(ninja_file, width=120)
-        ninja_define_ebpf_compiler(nw, arch=arch, compiler=ebpf_compiler)
```

Comment:
cc @lebauce since I believe you were the one to introduce this in https://github.com/DataDog/datadog-agent/pull/29862